### PR TITLE
Fix and add translations (ja-JP)

### PR DESCRIPTION
**Pull request by yudete**
This PR adds and fixes Japanese (ja-JP) translations.

### CODE REVIEW

File: Files/Strings/ja-JP/Resources.resw [hunk 1]
```diff
@@ -910,13 +910,13 @@
     <value>ターミナルアプリ</value>
   </data>
   <data name="SettingsAboutSupportUs.Text" xml:space="preserve">
-    <value>応援する</value>
+    <value>支援する</value>
   </data>
   <data name="SettingsAboutSupportUsDescription.Text" xml:space="preserve">
-    <value>PayPalで応援する</value>
+    <value>PayPalで支援する</value>
```
Comment (siketyan) [inline Files/Strings/ja-JP/Resources.resw:916]: Maybe:
```suggestion
    <value>PayPal で支援する</value>
```

As you did at: https://github.com/files-community/Files/pull/2848/files#diff-5bce934836368afdd400bfe79c43711d851b1137c2ec59d51b1b3ea1ca8e717eR259

Comment (yudete) [reply Files/Strings/ja-JP/Resources.resw]: Thank you! I applied.

**Review by tsvietOK — CHANGES_REQUESTED** [review]
Don't change Resources file. To add translation change required file in MultilingualResources folder. This file will be regenerated automatically.

**Comment by yudete** [comment]
> Don't change Resources file. To add translation change required file in MultilingualResources folder. Resources file will be regenerated automatically.

Now I undid changes in `Resources.resw` and move translations to `Files.ja-JP.xlf`.

File: Files/MultilingualResources/Files.ja-JP.xlf [hunk 2]
```diff
@@ -1274,31 +1274,31 @@
         </trans-unit>
         <trans-unit id="FolderItemAutomation" translate="yes" xml:space="preserve">
           <source>Folder</source>
-          <target state="needs-review-translation" state-qualifier="tm-suggestion">フォルダー</target>
+          <target state="needs-review-translation" state-qualifier="tm-suggestion">フォルダ</target>
         </trans-unit>
         <trans-unit id="RecycleBinItemAutomation" translate="yes" xml:space="preserve">
           <source>Recycle bin item</source>
-          <target state="new">Recycle bin item</target>
+          <target state="new">ごみ箱の項目</target>
```
[the remark this answers is not in the document]
Comment (yudete) [reply Files/MultilingualResources/Files.ja-JP.xlf]: Applied.

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**